### PR TITLE
[Payum] Add missing model interfaces

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | -----
| Branch?         | 1.2
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | 
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/PayumBundle/Model/GatewayConfigInterface.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\PayumBundle\Model;
+
+use Payum\Core\Model\GatewayConfigInterface as BaseGatewayConfigInterface;
+use Sylius\Component\Resource\Model\ResourceInterface;
+
+interface GatewayConfigInterface extends BaseGatewayConfigInterface, ResourceInterface
```
Comment (Zales0123) [inline src/Sylius/Bundle/PayumBundle/Model/GatewayConfigInterface.php]: I suppose it should be now implemented by `Sylius\Bundle\PayumBundle\Model\GatewayConfig`?

File: src/Sylius/Bundle/PayumBundle/Model/PaymentSecurityTokenInterface.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\PayumBundle\Model;
+
+use Payum\Core\Security\TokenInterface;
+use Sylius\Component\Resource\Model\ResourceInterface;
+
+interface PaymentSecurityTokenInterface extends ResourceInterface, TokenInterface
```
Comment (Zales0123) [inline src/Sylius/Bundle/PayumBundle/Model/PaymentSecurityTokenInterface.php]: Same here with `Sylius\Bundle\PayumBundle\Model\PaymentSecurityToken`

File: src/Sylius/Bundle/PayumBundle/Model/GatewayConfigInterface.php (hunk 1 above)
Comment (GSadee) [reply src/Sylius/Bundle/PayumBundle/Model/GatewayConfigInterface.php]: Done 👍

File: src/Sylius/Bundle/PayumBundle/Model/PaymentSecurityTokenInterface.php (hunk 2 above)
Comment (GSadee) [reply src/Sylius/Bundle/PayumBundle/Model/PaymentSecurityTokenInterface.php]: Done 👍

**Comment by lchrusciel** [comment]
Thank you, Grzegorz! :tada:

**Pull request merged**